### PR TITLE
fix: remove typos on different pages

### PR DESCRIPTION
**Pull request by milljoniaer**
### How to review
- Open the website local in the browser
- Check the corrected words on the changed pages

### CODE REVIEW

**Comment by ghost** [comment]
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**satellytes.com-main**

:tada: Your build was successful! [See the Deploy preview here.](https://satellytescommain-jotypos.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/f215130d-15e3-4176-9252-e13166fe48de/sites/d0e52c9f-427b-4c94-ba6b-8a84791ebbf1/builds/f3795d60-aff1-4fe5-b739-e2cd2e8691e4/details)

:clock1: Build time: **4m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :large_orange_diamond: 70   |
| Accessibility |   :green_heart: 100   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 85   |

:link: [View full report](https://build-f3795d60-aff1-4fe5-b739-e2cd2e8691e4.gtsb.io/reports/lighthouse/index.html)

File: src/locales/en/translations.json [hunk 1]
```diff
@@ -85,14 +85,14 @@
     "info-link": "<0>Use our contact-form or write an e-mail to <1>beep@satellytes.com</1></0>",
     "info": "Use our contact-form or write an e-mail to beep@satellytes.com",
     "name": "Name",
-    "email": "E-mail-adresse",
+    "email": "E-mail-adress",
```
Comment (fischeversenker) [inline src/locales/en/translations.json]: `Adresse` is `address` in English (double d, double s). Also I'm not sure about the second hyphen (`-`). Personally I'd remove it. Maybe you can clarify with Eric?
```suggestion
    "email": "E-mail address",
```

File: src/locales/en/translations.json [hunk 2]
```diff
@@ -85,14 +85,14 @@
...
     "message": "Your message to us",
     "mandatory-field": "mandatory field",
     "error": {
       "name": "Your name is missing",
       "message": "Your message is missing",
       "email": "Your e-mail is missing",
-      "email-unknown": "Something is wrong with the email-adresse"
+      "email-unknown": "Something is wrong with the e-mail-adress"
```
Comment (fischeversenker) [inline src/locales/en/translations.json]: Same here. It's `address`... and I would drop that second hyphen. However, I very much like the `e-mail` (as opposed to `email`).
```suggestion
      "email-unknown": "Something is wrong with the e-mail address"
```

File: src/pages/contact.tsx [hunk 3]
```diff
@@ -52,8 +52,8 @@ const ContactPage: React.FC<ContactPageProps> = (props: ContactPageProps) => {
               <SubTitle>E-Mail</SubTitle>
               <Trans i18nKey="contact.info-link">
                 <Text style={{ marginBottom: '40px' }}>
-                  Nutzen Sie unser Koaktformular oder schreiben Sie uns eine
-                  E-Mail an
+                  Nutzen Sie unser Kontaktformular oder schreiben Sie uns eine
+                  E-Mail an&nbsp;
```
Comment (fischeversenker) [inline src/pages/contact.tsx]: Did you ask yourself why the added `&nbsp;` is apparently not necessary for English? I'd rather not add this here. Let's figure out why it works in English and try to apply that same solution to German.

File: src/templates/career.tsx [hunk 4]
```diff
@@ -132,7 +132,7 @@ const CareerPage = ({
           <GridItem xs={12}>
             <Trans i18nKey={'career.action.alternative'}>
               <InfoText>
-                Oder schicke deine Bewerbung an:{' '}
+                Oder schicke deine Bewerbung an:
```
Comment (fischeversenker) [inline src/templates/career.tsx:135]: Good catch. It seems like this is indeed not needed anymore 👌 
This seems to have been an alternative solution to the `&nbsp;` that you proposed. Maybe you were wondering why you were able to remove this here without breaking the whitespace but at the same time had to introduce it for the contact form?
Changes like this should ring an alarm bell 🔔

File: data/pages/services-de.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ Banken- und Automobilbranche. Angular hat sich für derartige Applikationen zur
 bringen tiefgreifende Erfahrung mit dieser Technologie mit und liefern Ihnen eine maßgeschneiderte Lösung zu jedem
 Problem.
 
-Wir sind zudem sehr erfahren in der Einführung ganzheitlicher Plattformen –beispielsweise einer Infrastruktur aus
+Wir sind zudem sehr erfahren in der Einführung ganzheitlicher Plattformen – beispielsweise einer Infrastruktur aus
```
Comment (fischeversenker) [inline data/pages/services-de.md:19]: Nice! Thanks for noticing and fixing this 👍

**Review by fischeversenker — COMMENTED** [review]
Thanks for this nice PR 🙌 

I left a few comments, mostly regarding the white-space changes. Please have a look 👌

File: src/templates/career.tsx (hunk 4 above)
Comment (milljoniaer) [reply src/templates/career.tsx:135]: Thanks for the comment. I think I found the reason for this. The right translation is now used in the contact component. I changed the name of the german translation

**Review by fischeversenker — APPROVED** [review]
Thanks for the updates :pray:
This looks great now. Feel free to merge :sparkles:

It will be deployed automatically so after the merge you will see your changes on the live website soon 🎉

**Pull request merged**